sunkit_pyvista/plotter.py

```diff
@@ -199,6 +201,8 @@ def plot_coordinates(self, coords, radius=0.05, **kwargs):
             point_mesh = pv.Spline(points)
         else:
             point_mesh = pv.Sphere(radius=radius, center=points[0])
+        color = kwargs.get('color', np.nan)
```

Comment:
What does setting the default to nan do?

Reply:
`field_arrays` are converted to numpy float and `None` isn't converted but stored as a string `'None'`, I thought nan would be better indicating it has no color